### PR TITLE
 feat: Implementation of CRUD functionality for availability model

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/assesment/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Generated by Django 3.2.5 on 2023-02-21 03:09
```
Comment (Raam124) [inline apps/assesment/migrations/0001_initial.py:1]: why there are migrations files in these changes?

File: apps/assesment/views.py [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+from rest_framework import viewsets
+from .models import Avalability
+from .serializers import AvalabilitySerializer
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
 
-# Create your views here.
+
+@csrf_exempt
```
Comment (Raam124) [inline apps/assesment/views.py:10]: we're not using function-based views. use class-based views

File: apps/dispensary/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
+# Generated by Django 3.2.5 on 2023-02-21 03:09
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```
Comment (Raam124) [inline apps/dispensary/migrations/0001_initial.py:7]: Again where do these migrations come from? did we already merge all migrations?

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/assesment/views.py]: Went through the documentation and changed the code for implementation of class based views

Comment (Raam124) [reply apps/assesment/views.py]: API view is not suitable here. use the simplest one

File: apps/assesment/migrations/0001_initial.py (hunk 1 above)
Comment (Raam124) [reply apps/assesment/migrations/0001_initial.py:1]: @Kavindu-248

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/migrations/0001_initial.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/assesment/migrations/0001_initial.py:1]: My mistake on the migrations , yea its already merged

File: apps/assesment/views.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/assesment/views.py]: Implemented with ModelViewSet

File: apps/dispensary/migrations/0001_initial.py (hunk 3 above)
Comment (Kavindu-248) [reply apps/dispensary/migrations/0001_initial.py:7]: Yea migrations applied to dev this was a mistake locally  I will delete these files from this branch

Comment (Raam124) [reply apps/dispensary/migrations/0001_initial.py]: do not ever delete any migration files

**Pull request closed without merging**